packages/plugin-analytics/src/events/_start.ts

```diff
@@ -0,0 +1,39 @@
+import { logger } from 'robo.js'
+import { BaseEngine, setAnalytics } from '../utils/analytics'
+import { GoogleAnalytics } from '../utils/GoogleAnalytics'
+import { ManyEngines } from '../utils/ManyEngines'
+import { PlausibleAnalytics } from '../utils/PlausibleAnalytics'
+
+interface PluginOptions {
+	engine?: BaseEngine
+}
+export let pluginOptions: PluginOptions = {}
+
+export default (_client: unknown, options: PluginOptions) => {
+	pluginOptions = options ?? {}
+
+	if (pluginOptions.engine) {
+		setAnalytics(pluginOptions.engine)
+		return
+	}
+
+	const GAnalytics = process.env.GOOGLE_ANALYTICS_MEASURE_ID
```

Comment:
Please use `if/else if/else` for this to avoid several returns and throw an error instead of error log.

```ts
if (GAnalytics && PAnalytics) {
  // ...
} else if (GAnalytics) {
  // ...
} else if (PAnalytics) {
  // ...
} else {
  throw new Error('No Analytics Engine, please provide one, see https://robojs.dev/plugins/analytics to learn how.')
}
```